Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -47,6 +48,8 @@ class RenderFlexibleBox : public RenderBlock {
     RenderFlexibleBox(Document&, RenderStyle&&);
     virtual ~RenderFlexibleBox();
 
+    using Direction = BlockFlowDirection;
```

Comment:
I'm not convinced this alias is useful anymore. I would just use `BlockFlowDirection` directly. Easier to search.

Reply:
I personally don't like how the return type of a function like `crossAxisDirection` would be `BlockFlowDirection` since that's not really precise even though the values are equivalent. At the same time it's not a deal breaker for me so we can change it if we would really like to